docs/tutorials/upgrading-ev3dev-using-shell

```diff
@@ -0,0 +1,24 @@
+---
+title: Upgrading ev3dev Using Shell
+subject: Upgrading
```

Comment:
I'm not sure that "upgrading" is a broad enough topic name. Maybe "core" or "system"?